tests/unittests/functionality/test_google_oauth.py

```diff
@@ -21,9 +21,9 @@ def test_user_already_logged_in(self):
             register(self.app, 'email@gmail.com', 'test')
             logout(self.app)
             login(self.app, 'email@gmail.com', 'test')
-            self.assertTrue('Open Event' in self.app.get('/gCallback/?state=dummy_state&code=dummy_code',
+            self.assertTrue('Open Event' in self.app.get('/gCallback/?state=dummy_state&code=dummy_code&test=1',
```

Comment:
Any suggestions? Here is sample of test case would be created

````
    def test_user_already_logged_in(self):
        """If the user is already logged in then on clicking 'Login with Google' the user details should be updated"""
        with app.test_request_context():
            register(self.app, 'email@gmail.com', 'test')
            logout(self.app)
            login(self.app, 'email@gmail.com', 'test')
            # an exception will be thrown since there will be no data from user
            self.assertRaises('Open Event' in self.app.get('/gCallback/?state=dummy_state&code=dummy_code',
                                                         follow_redirects=True).data)
````